etc/buildsys/fvconf.mk

```diff
@@ -124,13 +124,14 @@ ifeq ($(HAVE_OPENCV),1)
   CFLAGS_OPENCV      = -DHAVE_OPENCV -Wno-deprecated-declarations \
 		       $(shell $(PKGCONFIG) --cflags 'opencv$(OPENCV_VERSION_SUFFIX)') \
 		       $(CFLAG_W_NO_UNUSED_LOCAL_TYPEDEFS)
+  LDFLAGS_OPENCV     = $(shell $(PKGCONFIG) --libs-only-L 'opencv$(OPENCV_VERSION_SUFFIX)')
```

Comment:
Why `--libs-only-L`? I would have expected this to be `--libs`.

Reply:
well... I used `--libs-only-L` because the individual `-l` libs are specified explicitly further below and seem to work as they are.

Reply:
Can you use `-l` and remove the list instead?

Reply:
I think someone did that because `pkgconfig --libs` simply lists all opencv libs, which may often be more than you actually need

Reply:
Makes sense!